app/templates/view_supplier_users.html

```diff
@@ -87,7 +87,7 @@
 
       {% call summary.field() %}
       {% if item.active %}
-        {% if current_user.has_role('admin') %}
+        {% if current_user.has_role('admin') and not item.personalDataRemoved %}
```

Comment:
Could you have the situation where a user was active, but also had their personal data removed? If not, this second conditional might be extra.